test/Reachability/solve_continuous.jl

```diff
@@ -15,17 +15,21 @@ A = [ 0.0509836  0.168159  0.95246   0.33644
 X0 = BallInf(ones(4), 0.1)
 
 # default options (computes all variables)
-s = solve(InitialValueProblem(LinearContinuousSystem(A), X0), :T=>0.1)
+s = solve(InitialValueProblem(LinearContinuousSystem(A), X0),
```

Comment:
i would prefer to keep the "default options" test and add the test where you specifically give `project_reachset` separately, what do you think?

Reply:
I can remove the option after merging #486 again.